// This file is part of DynExp.

syntax = "proto3";

import public "Common.proto";

package DynExpProto.NetworkDigitalIn;

message SampleMessage
{
	bool Value = 1;
}

service NetworkDigitalIn
{
	rpc GetAsync (DynExpProto.Common.VoidMessage) returns (SampleMessage) {}
	rpc GetSync (DynExpProto.Common.VoidMessage) returns (SampleMessage) {}
}